Firebase/Messaging/FIRMessaging.m

```diff
@@ -215,8 +212,10 @@ - (void)dealloc {
 #pragma mark - Config
 
 + (void)load {
+  [FIRApp registerLibrary:(id<FIRLibrary>)self
```

Comment:
Same as above, the cast shouldn't be necessary if we're passing in the class.

Reply:
Done.